packages/server-wallet/src/wallet/index.ts

```diff
@@ -138,8 +138,13 @@ export class Wallet implements WalletInterface {
   async closeChannel(_args: CloseChannelParams): SingleChannelResult {
     throw 'Unimplemented';
   }
+
   async getChannels(): MultipleChannelResult {
-    throw 'Unimplemented';
+    const channels = await Channel.transaction(async tx => await Store.getChannels(tx));
+    return {
+      channelResults: channels.map(channel => channel.channelResult),
+      outbox: [],
+    };
```

Comment:
A transaction isn't needed here, since `getChannels` only involves a single query.

If you make the change suggested below, this can then be

```suggestion
    return { channelResults: await Store.getChannels(), outbox: [] };
```

Reply:
90498f815